app/main.py

```diff
@@ -1,6 +1,29 @@
+import os
+import requests
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    base_url = "https://api.weatherapi.com/v1/current.json?"
+    api_key = os.getenv("API_KEY")
+
+    city = "Paris"
+
+    try:
+        url = base_url + "key=" + api_key + "&q=" + city
+        response_json = requests.get(url).json()
+    except TypeError:
+        raise TypeError(
+            "Set your environment variable API_KEY "
+            "in terminal and run this script again"
+        )
```

Comment:
I think it's better to check if  API_KEY  in your env variables use `os.getenv("API_KEY")`